src/main/java/easy/gc_coffee_api/dto/MenuResponseDto.java

```diff
@@ -0,0 +1,25 @@
+package easy.gc_coffee_api.dto;
+
+import easy.gc_coffee_api.entity.Thumnail;
+import easy.gc_coffee_api.entity.common.Category;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class MenuResponseDto {
+    private String name;
+    private int price;
+    private Category category;
+    private Thumnail thumnail;
```

Comment:
너무 잘 작성해주셨는데 dto에 Thumnail 바로 나가면 안되서 아이디로 매핑해서 반환해주면 좋을 것 같아요!!
```suggestion
    private Long thumnailId;
```

Reply:
감사합니다. 확인하겠습니다.